README.spanish.md

```diff
@@ -97,112 +114,122 @@
 
 ## ![✔] 2.1  Usa Async-Await o promesas para manejo de errores asíncronos
 
-**TL;DR:** El manejo de errores asincrónicos en el estilo de callback es probablemente la manera más rápida de ir al infierno (a.k.a, pyramid of doom o pirámide de la perdición). El mejor regalo que le puedes dar a tu código es utilizar una biblioteca de promesas seria o async-await que proporcione una sintaxis de código muy compacta y familiar como try-catch
+**TL;DR:** El manejo de errores asíncronos en el estilo de callback es probablemente la manera más rápida de ir al infierno (a.k.a, pyramid of doom o pirámide de la perdición). El mejor regalo que le puedes dar a tu código es utilizar una biblioteca de promesas seria o async-await que proporciona una sintaxis de código muy compacta y similar al try-catch
 
-**De lo contrario:** El estilo de callback de Node.JS, function (err, response), es una forma prometedora de código no mantenible debido a la combinación de manejo de errores con código accidentado, anidación excesiva y patrones de codificación incómodos
+**De lo contrario:** El estilo de callback de Node.JS, function (err, response), es una forma prometedora de código no mantenible debido a la combinación de manejo de errores con código normal, anidación excesiva y patrones de codificación incómodos
 
-🔗 [**Leer más: evitar callbacks**](/sections/errorhandling/asyncerrorhandling.spanish.md)
+🔗 [**Leer más: evitar callbacks**](./sections/errorhandling/asyncerrorhandling.spanish.md)
 
 <br/><br/>
 
 
-## ![✔] 2.2 Usa solo el objeto Error incorporado
+## ![✔] 2.2 Usa solo el objeto Error nativo
```

Comment:
Incorporado isn't really like native, and also the title in the corresponding section is translated as i did here